lib/number.js

```diff
@@ -4,7 +4,8 @@ module.exports = function (options, input, {
   allowNaN = options.number.hasOwnProperty('allowNaN') ? options.number.allowNaN : true,
   allowInfinity = options.number.hasOwnProperty('allowInfinity') ? options.number.allowInfinity : true,
   allowFindInString = options.number.hasOwnProperty('allowFindInString') ? options.number.allowFindInString : true,
-  allowDecimals = options.number.hasOwnProperty('allowDecimals') ? options.number.allowDecimals : true
+  allowDecimals = options.number.hasOwnProperty('allowDecimals') ? options.number.allowDecimals : true,
```

Comment:
We could almost deprecate this, as `withPrecision === 0` would almost us the same result (integer vs float).